frontend/src/features/landing_page/components/GeoMap.tsx

```diff
@@ -47,4 +50,29 @@ export const GeoMap = () => {
       {markers}
     </MapContainer>
   );
+
+  function handleClick() {
+    setIsVisible(true);
+  }
+
+  let content;
+
+  if (isVisible) {
+    content = map;
+  } else {
+    content = (
+      <div className="text-center" id="placeholder">
+        <img
+          src="/permaplant-map-placeholder.png"
+          alt="PermaplanT Map"
+          className="mb-2 hover:cursor-pointer"
+          onClick={handleClick}
+        />
+        <i>{t('geomap:hint')}</i>
+        <br />
```

Comment:
```suggestion
        <span className="my-2 flex-1 shrink-0 grow text-sm italic">{t('geomap:hint')}</span>
```
The `<i>` tag should not be used here. It has the semantic meaning of reading something in a different voice or mood such that it is different from the rest of the text.
See here for the exact spec https://html.spec.whatwg.org/#the-i-element

Also, why do we need the `<br>`?

I think smaller would look a little better. The other styles are such that it does not overflow the parent on smaller screens.

Reply:
I forgot to delete the `<br>`. Originally there was a dedicated button to load the map, but I later replaced it with the whole map div as the button.